internal/pkg/probe/probe.go

```diff
@@ -65,3 +67,35 @@ func (m *deviceScanner) ScanDevices(ctx context.Context, log logr.Logger) ([]str
 	}
 	return paths, nil
 }
+
+// GetUnfomattedDevices retrieves devices that are unformatted.
+func (m *deviceScanner) ScanAvailableDevices(ctx context.Context, log logr.Logger) (*block.DeviceList, error) {
```

Comment:
@croomes @jmclong any concerns if I combine this logic into `ScanDevices` method?

Reply:
No, I think that would make more sense? It looks like ScanDevices is only used
in one place now, and looks like it could also ignore formatted?

Reply:
Yeah, I think ScanDevices should just ignore formatted that aren't owned by the VG